cleanup/action.yml

```diff
@@ -1,11 +1,14 @@
 ---
 name: 'Clean workspace'
-description: 'Clean workspace directory after previous workflow run'
+description: 'Clean workspace directory by making an empty detached commit'
 runs:
   using: 'composite'
   steps:
     - run: |
         set -e
         shopt -s dotglob
-        rm -rf ${{ github.workspace }}/*
+        git checkout -f \
+          $(git -c user.name=TarantoolBot -c user.email=bot@tarantool.io commit-tree \
```

Comment:
Looks like you should break the line after `user.email=bot@tarantool.io`.

Reply:
okay